### DISCUSSION TITLE
Microsoft SQL Server / MSSQL database service integration

### DISCUSSION THREAD

**Discussion by marko-hologram** [coollabsio/coolify#2182] [Database Requests]
Hey, first of all thank you for making Coolify!

I decided to give it another try and I'm loving it so far. Unfortunately, for types of projects I'm working on, I do require MSSQL lot of the time because I'm deploying some .NET Core applications. There is a possibility for me to deploy MSSQL using Docker Compose file, but it being an out of the box supported service is just better UX. I would love it to have same type of experience as Postgres, Redis or MySQL in Coolify.

Here are some links regarding MSSQL and Docker:

- https://learn.microsoft.com/en-us/sql/linux/sql-server-linux-docker-container-deployment?view=sql-server-ver16&pivots=cs1-bash
- https://learn.microsoft.com/en-us/sql/linux/quickstart-install-connect-docker?view=sql-server-ver16&tabs=cli&pivots=cs1-bash
- https://docs.docker.com/samples/ms-sql/ -- docker compose example with appropriate healthcheck command
- https://hub.docker.com/_/microsoft-mssql-server

I tried looking at previous issues/discussions for this, but only thing I found is this https://github.com/coollabsio/coolify/issues/379 and it doesn't seem to be relevant anymore.

---

Only problematic thing here might be licensing? Would that be a potential issue? https://www.microsoft.com/en-us/sql-server/sql-server-2019-pricing

There are some free versions of MSSQL, but I find the licensing confusing to be honest.

**Comment by jellydn** [comment]
Hi @marko-hologram, have you found out the solution yet?

**Reply by marko-hologram** [reply]
I haven't really used Coolify since then. Could probably be done with a custom docker compose file, but I didn't really dig into it at all.

**Comment by marco-derek** [comment]
Would love for this to be added. To build onto this my recommendation is that its integrated the same way that the other databases are, (like mysql) but by default the env var for ``ACCEPT_EULA`` is set to ``N``. This way a user has to accept the eula themselves.

Currently you are able to just slap it in as a docker image, but biggest caveats rn are we have to manually manage backups for the database myself nor do i get the nice SSL implementation that @peaklabs-dev added to coolify recently.

**Comment by sync-adm** [comment]
+1
